src/commands/debugTestCommand.ts

```diff
@@ -1,9 +1,21 @@
+import { relative } from "path";
+import { workspace, WorkspaceFolder } from "vscode";
+
 import { getTestRunner } from "../runners/TestRunnerFactory";
 
-async function debugTest(rootPath, fileName, testName) {
+async function debugTest(
+  rootPath: WorkspaceFolder,
```

Comment:
This is not a `WorkspaceFolder` object, it's a string

![image](https://user-images.githubusercontent.com/41467575/43043754-2014204e-8d9a-11e8-9cd0-3cee2d4ddc21.png)